specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1505 @@
+        "message": {
+          "description": "Error message.",
+          "type": "string"
+        },
+        "target": {
+          "description": "Error target.",
+          "type": "string"
+        },
+        "details": {
+          "description": "Error details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorDetails"
+          }
+        },
+        "innerError": {
+          "description": "Inner error message.",
+          "type": "string"
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "Common error details representation.",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "target": {
+          "description": "Error target.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message.",
+          "type": "string"
+        }
+      }
+    },
+    "ErrorResponseBody": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorResponseBody"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "IPTrafficProperties": {
+      "type": "object",
+      "properties": {
+        "sourceIps": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "destinationIps": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "sourcePorts": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "destinationPorts": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "protocols": {
+          "$ref": "#/definitions/NetworkProtocols"
+        }
+      }
+    },
+    "NetworkProtocols": {
+      "enum": [
+        "TCP",
+        "UDP",
+        "ICMP"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "NetworkProtocols"
+      }
+    },
+    "NetworkVerifier": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/NetworkVerifierProperties"
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "location": {
+          "type": "string"
```

Comment:
if you are modelling these resources as Tracked then they all need to support patch for tags atleast

Reply:
Used the TrackedResource type. Will add support for Patching later.

Reply:
Supporting PATCH for tags is mandatory : 

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#patch-resource](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#patch-resource)

Reply:
its also flagged in the errors in your linter tests : 

[https://github.com/Azure/azure-rest-api-specs/runs/8140755343](https://github.com/Azure/azure-rest-api-specs/runs/8140755343)

Reply:
I added the PATCH api for our TrackedResource.